<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Retour">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>401</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="seuil">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Assigner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Text_sh">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Text_sb">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seuil haut (en pourcent)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seuil bas (en pourcent)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lin_seuil">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Push_Seuil">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Assigner</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chainage</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Param">
    <property name="title">
     <string>&amp;Param</string>
    </property>
    <widget class="QMenu" name="menu_Matrice">
     <property name="title">
      <string>&amp;Matrice</string>
     </property>
     <addaction name="action_1"/>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
    </widget>
    <widget class="QMenu" name="menu_Seuillage">
     <property name="title">
      <string>&amp;Seuillage</string>
     </property>
     <addaction name="actionHysteresis"/>
     <addaction name="actionAutre"/>
    </widget>
    <addaction name="menu_Matrice"/>
    <addaction name="menu_Seuillage"/>
    <addaction name="action_Charger"/>
    <addaction name="actionBiDirection"/>
    <addaction name="actionSauvegarder"/>
   </widget>
   <addaction name="menu_Param"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_1">
   <property name="text">
    <string>&amp;Prewitt</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>&amp;Sobel</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>&amp;Kirsch</string>
   </property>
  </action>
  <action name="action_Charger">
   <property name="text">
    <string>&amp;Charger</string>
   </property>
  </action>
  <action name="actionBiDirection">
   <property name="text">
    <string>BiDirection</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionHysteresis">
   <property name="text">
    <string>Hysteresis</string>
   </property>
  </action>
  <action name="actionAutre">
   <property name="text">
    <string>Seuil max</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignMatrix2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignMatrix3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Charger</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadPictures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSauvegarder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Sauvegarder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBiDirection</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BiDirection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seuil</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignSeuil()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Push_Seuil</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignNb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHysteresis</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignHyst()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAutre</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>assignMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ischain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>assignMatrix1()</slot>
  <slot>assignMatrix2()</slot>
  <slot>assignMatrix3()</slot>
  <slot>loadPictures()</slot>
  <slot>Sauvegarder()</slot>
  <slot>BiDirection()</slot>
  <slot>assignSeuil()</slot>
  <slot>assignHyst()</slot>
  <slot>assignMax()</slot>
  <slot>assignNb()</slot>
  <slot>ischain()</slot>
 </slots>
</ui>
